pages/content/posts/2022-q3-report/index.md

```diff
@@ -0,0 +1,178 @@
+
+It is therefore essential for the web developer community that MDN continues to be maintained and improved, and the core team continues to be supported in reviewing PRs from the contributor community.
+
+## The contribution of Open Web Docs
+
+Open Web Docs is the most significant organization contributing to MDN content maintenance. Of the multiple MDN organization repositories supported by Open Web Docs, the busiest is **mdn/content**, which contains the source for MDN’s pages. This repository is extremely active:
+
+- mdn/content is the 5th most active repository on all of GitHub, as measured by the number of commits/month according to [https://git-pulse.github.io/snapshots/](https://git-pulse.github.io/snapshots/).
+- In Q3 2022, 2,607 PRs were merged into mdn/content from 535 unique contributors.
+
+This report will demonstrate OWD’s contribution to the day-to-day maintenance of mdn/content with two metrics:
+
+- **Quarterly PRs merged to the mdn/content repository.**
+- **Quarterly number of reviews performed on each merged PR to the mdn/content repository.**
+
+Both these metrics contain contributions by OWD team members, other organizations that help maintain the repository, and volunteer contributors. Contribution metrics are divided into the following groups:
+
+- OWD staff
+- W3C staff (this is actually just Mike Smith)
+- Mozilla staff
+- Other (mostly volunteers, but also people paid by other organizations to work on MDN)
+
+As [@queengooborg](https://github.com/queengooborg) works for both OWD and Mozilla, her contributions have been evenly split between each organization.
+
+### PRs merged to mdn/content
+
+All merged PRs: 2607
+OWD: 284.5
+W3C: 42
+Mozilla: 136.5
+Other: 2144
+![Percentage pie chart. OWD: 10.9%, Mozilla: 5.2%, W3C: 1.6%, Other: 82.2%](prs.png)
+
+This demonstrates how, of organizations contributing to mdn/content as measured by PR volume, OWD is the biggest contributor. It also shows that mdn/content gets a huge volume of contributions from volunteers: 82.2% of PRs in Q2 were “Other”, which is mostly volunteers.
+
+### Reviews of mdn/content PRs
+
+All merged PRs: 3090
+OWD: 1942
+W3C: 349
+Mozilla: 193
+Other: 603
+![Percentage pie chart. OWD: 62.9%, W3C: 11.3%, Mozilla: 6.2%, Other: 19.5%](reviews.png)
+
+The small OWD team provides the largest number of PR reviews to mdn/content, by a significant margin. OWD technical writers performed 63% of all reviews on merged PRs in the quarter.
+
+# Projects
+
+In the third quarter of 2022, OWD completed the projects listed below.
+
+## Modernization of JavaScript docs
+
+During the 2010s, JavaScript evolved substantially as a language. The MDN Web Docs JavaScript documentation dates back to ES3. The docs were completely revamped around 2015 (for ES6), however, since then the docs didn’t see a major update throughout. New features were documented and used latest best practices, but docs for older features mostly demonstrate best practices of the time.
```

Comment:
```suggestion
During the 2010s, JavaScript evolved substantially as a language. The MDN Web Docs JavaScript documentation dates back to ES3. The docs were completely revamped around 2015 (for ES6); however, since then, the docs didn’t see a major update throughout. New features were documented and used the latest best practices, but docs for older features mostly demonstrate the best practices of the time.

MDN code samples followed the same pattern. We updated them to follow ES5, but no more afterward. When reading them, they looked old school and cast doubt on the accuracy of the documentation.
```